### PR TITLE
Removes examples for non-exported functions

### LINKED ISSUE #1084: remove examples for non-exported functions
Examples for non-exported functions should be removed before the initial CRAN release and restored afterwards.
The Examples in question can be easily identified as they all contain getFromNamespace calls.

Remove them all in one commit (can be a squash commit of a PR) so that it can be easily reverted.

### PR DESCRIPTION
**Pull request by averissimo**
# Pull Request

Fixes #1084

### CODE REVIEW

File: R/module_nested_tabs.R [hunk 1]
```diff
@@ -32,68 +32,6 @@
 #' `srv_nested_tabs` returns a reactive which returns the active module that corresponds to the selected tab.
 #'
 #' @examples
```
Comment (donyunardi) [inline R/module_nested_tabs.R:34]:
```suggestion

```

Remove this line?

**Review by donyunardi — DISMISSED** [review]
Just missing one line that needs to be removed.
everything else looks good!

File: R/module_nested_tabs.R (hunk 1 above)
Comment (averissimo) [reply R/module_nested_tabs.R]: Oh shoot :sweat_smile:

**Review by m7pr — APPROVED** [review]
Checked that examples no longer exists in `@keywords internal` files and that there are no empty `@examples`.

**Review by donyunardi — APPROVED** [review]

**Pull request merged**